website/source/docs/guides/monitoring-telegraf.html.md

```diff
@@ -0,0 +1,299 @@
+
+**What to look for:** If `sys_bytes` exceeds 90% of `total_bytes`, if `mem.used_percent` is over 90%, or if
+`swap.used_percent` is greater than 0.
+
+### Garbage collection
+
+| Metric Name | Description |
+| :---------- | :---------- |
+| `consul.runtime.total_gc_pause_ns` | Number of nanoseconds consumed by stop-the-world garbage collection (GC) pauses since Consul started. |
+
+**Why it's important:** As mentioned above, GC pause is a "stop-the-world" event, meaning that all runtime threads are blocked until GC completes. Normally these pauses last only a few nanoseconds. But if memory usage is high, the Go runtime may GC so frequently that it starts to slow down Consul.
+
+**What to look for:** Warning if `total_gc_pause_ns` exceeds 2 seconds/minute, critical if it exceeds 5 seconds/minute.
+
+**NOTE:** `total_gc_pause_ns` is a cumulative counter, so in order to calculate rates (such as GC/minute),
+you will need to apply a function such as [non_negative_difference][].
+
+### File descriptors
+
+| Metric Name | Description |
+| :---------- | :---------- |
+| `linux_sysctl_fs.file-nr` | Number of file handles being used across all processes on the host. |
+| `linux_sysctl_fs.file-max` | Total number of available file handles. |
+
+**Why it's important:** Practically anything Consul does -- receiving a connection from another host, sending data between servers, writing snapshots to disk -- requires a file descriptor handle. If Consul runs out of handles, it will stop accepting connections. See [the Consul FAQ][consul_faq_fds] for more details.
+
+By default, process and kernel limits are fairly conservative. You will want to increase these beyond the defaults.
+
+**What to look for:** If `file-nr` exceeds 80% of `file-max`.
+
+### CPU usage
+
+| Metric Name | Description |
+| :---------- | :---------- |
+| `cpu.user_cpu` | Percentage of CPU being used by user processes (such as Consul). |
+| `cpu.iowait_cpu` | Percentage of CPU time spent waiting for I/O tasks to complete. |
+
+**Why they're important:** Consul is not particularly demanding of CPU time, but a spike in CPU usage might
+indicate too many operations taking place at once, and `iowait_cpu` is critical -- it means Consul is waiting
+for data to be written to disk, a sign that Raft might be writing snapshots to disk too often.
+
+**What to look for:** if `cpu.iowait_cpu` greater than 10%.
+
+### Network activity
+
+| Metric Name | Description |
+| :---------- | :---------- |
+| `net.bytes_recv` | Bytes received on each network interface. |
+| `net.bytes_sent` | Bytes transmitted on each network interface. |
+
+**Why they're important:** A sudden spike in network traffic to Consul might be the result of a misconfigured
```

Comment:
We should maybe add an RPC section for similar reasons. Just looking at bytes to the host might not cover all that we need. RPC is a more reliable indicator of client traffic.

Reply:
Makes sense, I added a block for the RPC count and RPC rate-limit indicators, which seem to be the right ones to look at. I'll tag you for a look